packages/cozy-client/src/__tests__/setup.js

```diff
@@ -1,4 +1,7 @@
 import Enzyme from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
+import fetch from 'whatwg-fetch'
```

Comment:
Shouldn't we add it to the package.json ?

Reply:
Done, thanks.